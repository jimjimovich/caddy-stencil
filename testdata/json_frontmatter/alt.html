{
    "footer": "Finally, a footer from <strong>JSON</strong>",
    "title": "Title from JSON",
    "template": "alternate"
}
<p>Just some <strong>HTML</strong>.</p>