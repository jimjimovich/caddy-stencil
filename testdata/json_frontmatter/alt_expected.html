<!DOCTYPE html>
<html>
    <head>
        <title>Title from JSON</title>
    </head>
    <body>
        
        <p>This is an alternate template</p>
        Finally, a footer from <strong>JSON</strong>
    </body>
</html>